web/core/build.gradle.kts

```diff
@@ -31,6 +31,7 @@ kotlin {
             dependencies {
                 implementation(project(":internal-web-core-runtime"))
                 implementation(kotlin("stdlib-js"))
+                implementation("org.jetbrains.kotlin-wrappers:kotlin-csstype:3.0.10-pre.283-kotlin-1.6.10")
```

Comment:
do we need this dependency in web/core?

I thought it's going to be added by users in their projects.

Reply:
Your understanding is 100% correct - just like I said at the meeting it's a fake PR - nobody is going to merge this one - it was just to illustrate so that we check whether we need this (and this dependency was added for test)

Reply:
got it, thanks! :) 
i forgot that it's a fake PR